lib/inkdrop-note-templates.js

```diff
@@ -4,17 +4,28 @@ const { Liquid } = require('liquidjs');
 const setupTemplates = async () => {
     const db = inkdrop.main.dataStore.getLocalDB()
     const TEMPLATE_NOTEBOOK_NAME = "_Templates";
-    const templateDocs = await db.books.findWithName(TEMPLATE_NOTEBOOK_NAME);
-    if (!templateDocs) {
-        console.log(`Not found ${TEMPLATE_NOTEBOOK_NAME} notebook`)
-        return [];
+    const TEMPLATE_TAG_NAME = "Template";
+
+    let templateNotes = Array();
+    // find notes in _Templates notebook.
+    const templateBook = await db.books.findWithName(TEMPLATE_NOTEBOOK_NAME);
+    if (templateBook) {
+        const notesInBook = await db.notes.findInBook(templateBook._id);
+        templateNotes = templateNotes.concat(notesInBook.docs);
+    }
+    // find notes with Template tag.
+    const templateTag = await db.tags.findWithName(TEMPLATE_TAG_NAME);
+    if (templateTag) {
+        const notesWithTag = await db.notes.findWithTag(templateTag._id);
+        templateNotes = templateNotes.concat(notesWithTag.docs);
     }
-    const { docs } = await db.notes.findInBook(templateDocs._id);
-    if (!docs) {
-        console.log(`Not found docs in ${TEMPLATE_NOTEBOOK_NAME}`);
+
+    if (!templateNotes) {
```

Comment:
`templateNotes` is `[]` by default. (it will be `true`.)
Check the length of `templateNotes`.
```suggestion
    if (templateNotes.length === 0) {
```

Reply:
Oops! Such a shame mistake from me. In Python, an empty list is regarded as False in comparison statement. Let me correct it.